my/fbmessenger/all.py

```diff
@@ -0,0 +1,48 @@
+from typing import Iterator
+from my.core import Res
+from my.core.common import Stats
+from my.core.source import import_source
+
+from .common import Message, _merge_messages
+
+
+src_export  = import_source(module_name=f'my.fbmessenger.export')
+src_android = import_source(module_name=f'my.fbmessenger.android')
+
+
+@src_export
+def _messages_export() -> Iterator[Res[Message]]:
+    from . import export
+    # ok, this one is a little tricky
+    # export.Message type is actually external (coming from fbmessengerexport module)
+    # so it's unclear how to make mypy believe/check that common.Message is a structural subtype of export.Message
+    # we could use runtime_checkable, but then it might also crash in runtime
+    # which feels somewhat mean if someone is only using fmbessenger.export module and needs its attributes only
+    # so perhaps it makes sense that the typecheck belongs here?
+    for m in export.messages():
+        # NOTE: just 'yield m' works and seems to type check properly
+        if isinstance(m, Exception):
+            yield m
+        else:
+            # however, this way it results in a nicer error (shows the missing Protocol attributes)
```

Comment:
@seanbreckenridge just fyi -- may be useful for you 
I guess later I'll just move this away to some guide (or maybe even `Protocol` documentation?)